### DISCUSSION TITLE
Trying to upgrade the ghostscript version included in AMI yum

### DISCUSSION THREAD

**Discussion by ardowz** [ImageMagick/ImageMagick#2495] [Help]
Currently running in an AMI, when I do a yum install ImageMagick, it comes with ghostscript 8.70 as a depedency. I've been trying to install ghostscript manually 9.20 but I can't seem to make `convert` or `analyze` use the newer library. Any thoughts in doing this?

**Comment by dlemstra** [comment]
It looks like AMI (no clue what that is though) is linking the old Ghostscript library instead of executing the binaries of Ghostscript. And how do you know the old version is being used?

**Reply by ardowz** [reply]
Woops, AMI is the server image from amazon. Yum's package lists is also referring to amazon's packages.
I know the old version is used because when I try to run `convert` against a pdf. I'm getting this error:

```
exception 'ImagickException' with message 'Postscript delegate failed `[ghostscript library 8.70] -sstdout=%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 '-sDEVICE=pngalpha' -dTextAlphaBits=4 -dGraphicsAlphaBits=4 '-r150x150' -dFirstPage=1 -dLastPage=1 '-sOutputFile=/tmp/magick-232997DXNPHbKiXvO%d' '-f/tmp/magick-23299wyzX6NId4srj' '-f/tmp/magick-23299Cht-VGFKtZmO'': Error: /undefined in findresource
Operand stack:
   --dict:19/28(L)--   C2_0   1   --dict:5/5(L)--   --dict:5/5(L)--   Arial   --dict:11/12(ro)(G)--   --nostringval--   CIDFontObject   --dict:7/7(L)--   --dict:7/7(L)--   Adobe-Identity
```

It says the ghostscript 8.70

**Reply by dlemstra** [reply]
Not sure what the package name is but you probably need to also update gs-lib.

**Reply by ardowz** [reply]
I tried installing ghostscript 9.20 manually, but whenever I run the `convert` I will be met with the same issue, and have the same library stated there. So I'm wondering if theres a way to make the yum installed `ImageMagick` to point to the newer `gs-lib` I installed manually?
